discord/ui/view.py

```diff
@@ -293,31 +352,40 @@ def add_item(self, item: Item) -> None:
         self.__weights.add_item(item)
 
         item._view = self
+        if hasattr(item, "items"):
+            item.view = self
         self.children.append(item)
+        return self
 
-    def remove_item(self, item: Item) -> None:
-        """Removes an item from the view.
+    def remove_item(self, item: Item | int | str) -> None:
+        """Removes an item from the view. If an int or str is passed, it will remove by Item :attr:`id` or ``custom_id`` respectively.
 
         Parameters
         ----------
-        item: :class:`Item`
-            The item to remove from the view.
+        item: Union[:class:`Item`, :class:`int`, :class:`str`]
+            The item, item :attr:`id`, or item ``custom_id`` to remove from the view.
```

Comment:
Would this point to `View.id`?
```suggestion
            The item, item ``id``, or item ``custom_id`` to remove from the view.
```